HomeWork/src/main/java/HomeWork6/Employee.java

```diff
@@ -0,0 +1,57 @@
+package HomeWork6;
+
+public class Employee {
+    private String fullName;
+    private String position;
+    private String email;
+    private String phoneNumber;
+    private int age;
+
+    public Employee(String username, String position, String email, String numberPhone, int age) {
```

Comment:
Why didn't you change 'numberPhone' in the constructor to 'phoneNumber'? 
The same question to  'username'.
The same questions to getters and setters. Please remember that if you change the field's name, you should also rename the constructor parameter if any, and getter + setter.